@builderio-bot look at latest PR feedback and fix anything you agree with. Be skeptical. Reply to every comment (directly on the comment thread of each comment) if you fixed it or not and why. also lint still failing pleae fix: 

> agentnative@ fmt:check /home/runner/work/agent-native/agent-native
> prettier --check --log-level warn .

[warn] templates/clips/desktop/src/styles.css
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.

--- a/templates/clips/desktop/src/styles.css
+++ b/templates/clips/desktop/src/styles.css
@@ -1514,7 +1514,9 @@ button {
 }
 
 @keyframes signin-spin {
-  to { transform: rotate(360deg); }
+  to {
+    transform: rotate(360deg);
+  }
 }
 
 .signin-pending-text {

